src/community/advanced-data-card.tsx

```diff
@@ -43,9 +46,13 @@ type CardContentProps = {
     subtitleLinesMax?: number;
     description?: string;
     descriptionLinesMax?: number;
+    href?: string;
+    to?: undefined;
```

Comment:
to shouldn't be undefined here.
You should use ExclusifyUnion here too, like you do in `AdvancedDataCardProps`